test/functional/tests/stress/test_kedr.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from datetime import timedelta
+
+from test_tools.kedr import Kedr, KedrProfile
+from api.cas import cas_module, installer, casadm
+from core.test_run import TestRun
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+
+@pytest.fixture(scope="module")
+def install_kedr():
+    with TestRun.step("Checking if kedr is installed"):
+        if not Kedr.is_installed():
+            with TestRun.step("Installing kedr"):
```

Comment:
This step is nested in other step. It shouldn't be done like this - You cas use simply TestRun.LOGGER.info() here.
Additionally please avoid putting steps in fixtures or other helpers methods. It is more readable when all steps are placed in test body. Please change it in the whole file.